### ISSUE TITLE
Provide a live demo instance

### ISSUE THREAD

**Issue by spapas** [BeaconCMS/beacon_demo#21]
It'd be great to provide a live instance of this demo (for example hosted on fly.io) so we can very quickly test it. The db could be hosted on the fly.io machine and the machine restarted every day so the state would be cleaned daily.

**Comment by leandrocp** [comment]
Hi @spapas we thought about it and eventually destroyed the demo instance that used to run on fly.io because people tend to abuse it and even create inappropriate content which can cause us legal troubles. Instead we've been thinking about creating a mix task `mix beacon.deploy [provider]` that generates the config as https://hexdocs.pm/beacon/deploy-to-flyio.html and automatically deploy to Fly.io. Sure not as straightforward as visiting a pre-deployed instance but not too bad as setting up from scratch.

**Comment by spapas** [comment]
@leandrocp makes sense ! Also if the sqlite3 support is there then it'd definitely be trivial to run the demo.
